### PR TITLE
[ml-service] Add nntrainer option 

### PR DESCRIPTION
**Pull request by gichan-jang**
Add nntrainer option to control dependency.
This temporarily solves the cyclic dependency problem when gcov is enabled.

### CODE REVIEW

**Review by niley7464 — APPROVED** [review]
LGTM!

**Review by again4you — APPROVED** [review]
LGTM!

**Review by jaeyun-jung — APPROVED** [review]

File: packaging/machine-learning-api.spec [hunk 1]
```diff
@@ -314,6 +315,7 @@ HTML pages of lcov results of ML API generated during rpm build
 %define enable_tizen_feature_check -Denable-tizen-feature-check=false
 %define enable_ml_service_check -Denable-ml-service=false
 %define enable_gcov -Denable-gcov=false
+%define enable_nntrainer -Denable-nntrainer=false
```
Comment (songgot) [inline packaging/machine-learning-api.spec:318]: How about default is true, and at line 336
%if 0%{?gcov}
%define enable_gcov -Denable-gcov=true
%define enable_nntrainer -Denable_nntrainer = false
%endif

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**